src/export/Export.cpp

```diff
@@ -336,21 +321,6 @@ bool Exporter::ExportTracks(
    return success;
 }
 
-bool Exporter::ProcessFromTimerRecording()
```

Comment:
I'm pleased to see this function unified with `Process`.  It was an ugly violation of open-closed to see this as part of the Exporter interface.

Fix the spelling in the commit comment: "became"